build/template-OneBranch-libsandsamples.yaml

```diff
@@ -20,14 +20,22 @@ steps:
   displayName: 'NuGet restore ${{ parameters.Solution }}'
   inputs:
     solution: ${{ parameters.Solution }}
+    vsVersion: 17.0
     msbuildArgs: '/t:restore'
     platform: ${{ parameters.BuildPlatform }}
     configuration: ${{ parameters.BuildConfiguration }}
 
+- template: /SharedComponents/OneBranch/Steps/Prepare.MSBuild.yml@templates
+  parameters:
+    msBuildVersion: 17.0
+    msbuildInstallationPath: c:\.vs
```

Comment:
does this need to be hardcoded?

Reply:
no, it can be a variable that is settable when stating the build